specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/capacityReservation.json

```diff
@@ -838,6 +838,12 @@
     "CapacityReservationUtilization": {
       "type": "object",
       "properties": {
+        "currentCapacity": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "The value provides the actual reserved count for the VM size which was provisioned sucessfully and for which the customer is getting billed for the Capacity Reservation.<br><br>Minimum api-version: 2022-08-01."
```

Comment:
The value provides the current capacity of the VM size which was reserved successfully and for which the customer is getting billed.<br><br>Minimum api-version: 2022-08-01.